LabExT/Movement/MoverNew.py

```diff
@@ -517,7 +525,12 @@ def move_absolute(
                 raise MoverError(
                     f"No {orientation} stage configured, but target coordinate for {orientation} passed.")
 
-            path_planning.set_stage_target(calibration, target)
+            current_stage_level = None
+            if calibration in self._lifted_stages:
+                current_stage_level = self.z_lift
```

Comment:
At this point your stages should be lifted anyhow.

I cannot see a reason why you would pass a z_level to the path planning. Doesn't path planning just move on a fixed z-level? If you start lifted, the path planning should stay on the lifted z-level anyhow, no?